cmd/ketch/autocompletion.go

```diff
@@ -0,0 +1,89 @@
+package main
```

Comment:
This is more of a nit pick than an actual issue, but I think auto_completion.go may be a better name for the file. I think it would be more in line with the naming convention already present in the repo.

Reply:
ok, renamed :)